cob_scan_unifier/src/scan_unifier_node.cpp

```diff
@@ -113,11 +114,22 @@ ScanUnifierNode::~ScanUnifierNode()
 void ScanUnifierNode::getParams()
 {
   std::vector<std::string> topicList;
+  bool point_cloud;
 
   if (pnh_.getParam("input_scans", topicList))
   {
     config_.input_scan_topics = topicList;
     config_.number_input_scans = config_.input_scan_topics.size();
+    if(!pnh_.hasParam("point_cloud"))
```

Comment:
maybe for better semantic naming call your parameter `publish_point_cloud` or something alike

also why is the parameter checked in the 
```
if (pnh_.getParam("input_scans", topicList))
{
```
section and not in the "root" of `getParams()`